{"uuid":"cdaf93cc-9d2f-47b6-ac19-ece5a731b4ad","historyId":"bd74d59f262e2933a98286c0e0935e2c","fullName":"ru.atc.fgislk.ppod.testcore.integrityControl.TestScenario.Test","labels":[{"name":"package","value":"ru.atc.fgislk.ppod.testcore.integrityControl.TestScenario"},{"name":"testClass","value":"ru.atc.fgislk.ppod.testcore.integrityControl.TestScenario"},{"name":"testMethod","value":"Test"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"fgislk-geo-tests"},{"name":"subSuite","value":"ru.atc.fgislk.ppod.testcore.integrityControl.TestScenario"},{"name":"host","value":"5CD6347KM7"},{"name":"thread","value":"14908@5CD6347KM7.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Test","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"ОШИБКА: отношение \"file_set.fct_file_set\" не существует\n  Position: 15","trace":"org.postgresql.util.PSQLException: ОШИБКА: отношение \"file_set.fct_file_set\" не существует\n  Position: 15\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2674)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2364)\r\n\tat org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:354)\r\n\tat org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:484)\r\n\tat org.postgresql.jdbc.PgStatement.execute(PgStatement.java:404)\r\n\tat org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:325)\r\n\tat org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:311)\r\n\tat org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:287)\r\n\tat org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:239)\r\n\tat ru.atc.fgislk.ppod.testcore.integrityControl.TestScenario.Test(TestScenario.java:42)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\r\n\tat org.testng.TestRunner.run(TestRunner.java:603)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\r\n\tat org.testng.TestNG.run(TestNG.java:1060)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1658913271069,"stop":1658913271169}